nomad/grapl-core.nomad

```diff
@@ -116,29 +134,30 @@ job "grapl-core" {
         min_healthy_time = "15s"
     }
 
-    group "dgraph-zero-leader" {
+    group "dgraph-zero-0" {
         network {
             mode = "bridge"
         }
 
-        task "dgraph-zero-leader" {
+        task "dgraph-zero" {
             driver = "docker"
 
             config {
-                image = "dgraph/dgraph:latest"
+                image = "dgraph/dgraph:${var.dgraph_tag}"
                 args = [
                     "dgraph",
                     "zero",
-                    "--my=localhost:5080",
+                    "--my", "localhost:5080",
```

Comment:
Not a must have, but worth noting we could totally do 
`localhost:${local.dgraph_zero_grpc_private_port_base}` 

there's a number of other places with an explicit 5080 in this PR that we could do similar string interpolation with; I won't highlight them all, just ctrl-f